src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -167,6 +167,7 @@ object AddDatasetMetadataToDeposit {
         case (Some(q), None,   Some(t)) => elem(s"dcterms:$q")(t)
         case (None,    Some(l),_      ) => elem(s"dc:relation")(l)
         case (None,    None,   Some(t)) => elem(s"dc:relation")(t)
+        case _                          => // TODO do nothing; this needs to be checked in the preconditions
```

Comment:
rephrase as 'preconditions should have reported this as an error', the real TODO is at line 32 and was mentioned in #15 (closed because its now in Jira)